get-donor-data/flatten-json-to-csvs.py

```diff
@@ -0,0 +1,203 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+description:
+version: 0.0.1
+created:
+author: Ed Nykaza
+dependencies:
+    *
+license: BSD-2-Clause
+TODO:
+* [] rewrite code to take advantage of parrallel processing, given that
+code takes so long to run
+* [] Make sure that .0 column headers do not exist
+* [] Add a check if the data was already processed, not to process
+"""
+
+# %% load in required libraries
+import pandas as pd
+import os
+import json
+from pandas.io.json import json_normalize
+import sys
+import numpy as np
+
+
+# %% user inputs (choices to be made to run code)
+securePath = "/tidepoolSecure/data/"
```

Comment:
Change to default path with overridable command line parameter.

Reply:
done